spec/schemas/prison-offender-events/prisoner.video-appointment.cancelled.yaml

```diff
@@ -0,0 +1,108 @@
+name: prison-offender-events.video-appointment.cancelled
```

Comment:
think this should be `prison-offender-events.prisoner.video-appointment.cancelled`

Reply:
I was aiming for `prison-offender-events.video-appointment.cancelled` - but think I've left a `prisoner` in one place by mistake.

Reply:
Now updated to be consistent event name.

Reply:
Updated to recommended name including `prisoner`